src/hooks/motionWidgets/stakingWidget/useRequiredStakeMessage.ts

```diff
@@ -0,0 +1,55 @@
+import Decimal from 'decimal.js';
+import { BigNumber } from 'ethers';
+import { useFormContext } from 'react-hook-form';
+import {
+  convertStakeToPercentage,
+  formatStakePercentage,
+  getStakeFromSlider,
+  SLIDER_AMOUNT,
+} from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
```

Comment:
I have put all these hooks into our top-level `hooks` directory, since that's the pattern we're using at the moment.

But really, all these hooks could live in their own `hooks` folder alongside the Staking Widget component. This would make the import address shorter, and also seems logical given that these hooks aren't going to be used outside of their respective components.